build-tests/prompt-agent.mdx

```diff
@@ -0,0 +1,134 @@
+---
+title: Prompt AI Agent
+description: "Use natural language to prompt our AI agent to generate new tests"
+icon: "language"
+---
+
+## Prompt the Agent to generate a specific test case
+
+If you want to translate a custom user flow into a test, you can have it AI generated from your prompt. Give us a short prompt and we deploy our `AI agent` to find the interactions leading towards your desired user flow.
+
+<Frame caption="Create new test case by prompting the AI agent, 07/2024">
+  <img
+    src="/images/build/create-test-prompting.png"
+    alt="AI agent informs about its progress"
+  />
+</Frame>
+
+While our `AI Agent` generates the steps of your test cases, it informs you about its progress.
+
+<Frame caption="AI Agent informs about the progress of test step generation, 07/2024">
+  <img
+    src="/images/build/AI-Agent-generating-steps.png"
+    alt="AI Agent informs about the progress of test step generation"
+  />
+</Frame>
+
+If the AI agent signals a failed step with a yellow alert, click on `repair test` to help the Agent finish the generation successfully.
+
+<Frame caption="AI agent needs help your input to finalize generation - click repair test, 05/2025">
+  <img
+    src="/images/build/repair-yellow-test.png"
+    alt="AI agent needs your input to finalize generation - click repair test"
+  />
+</Frame>
+
+Follow the instuction of the Agent to repair the test.
+
+<Frame caption="Follow the instructions of the AI agent to repair and finalize test generation, 05/2025">
+  <img
+    src="/images/build/repair-mode.png"
+    alt="Follow the instructions of the AI agent to repair and finalize test generation"
+  />
+</Frame>
+
+## Pre-prompted login test
+
+If your app has a login functionality that wasn't AI auto-generated during set-up, we have a pre-prompted login test case for you. You can trigger it after giving us test user credentials.
+
+Go to `environments` menu item in the left project menu bar and click on the `default environment` in the list. You can have separate login test for separate environment.
```

Comment:
```suggestion
Go to the `environments` menu item in the left project menu bar and click on the `default environment` in the list.
```

i don't think you can have separate logins? At least I don't think this is intended as it makes no sense.

Reply:
what you CAN have is separate CREDENTIALS per environment